src/admin-views/widgets/components/section.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Admin View: Widget Section Component.
+ *
+ * This component is different in that it calls other components!
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/section.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var string              $label       Title for the section. (optional)
+ * @var string              $description Description for the section. (optional)
+ * @var string              $classes     Classes to add to the section. (optional)
+ * @var string              $dependency  The dependency attributes for the control wrapper.
+ * @var array<string,mixed> $children    Child elements for the section.
+ */
+
+use Tribe__Utils__Array as Arr;
+
+$section_classes = array_merge( [ 'tribe-widget-form-control', 'tribe-widget-form-control--section' ], Arr::list_to_array( $classes, ' ' ) );
+
+?>
+<div
+	<?php tribe_classes( $section_classes ); ?>
+	<?php
+		// Not escaped - contains html ( data-attr="value")
+		echo $dependency;
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 5fbbf6e87deb06fd2fd2c5368a8efa1f</sub>